DPANIC statuscake-monitor Error on v2.1.5

We've moving to the stakater hosted chart running the latest `v2.1.5`, previously running `v2.0.14`.

The pod is going into `CrashBackoffLoop` with the error `DPANIC	statuscake-monitor	odd number of arguments passed as key-value pairs for logging`

I've removed all mentions of the previous version of the chart and CRD incase of any clash, but the error continues. 

Full Logs from startup;
```
I0517 18:11:06.292973       1 request.go:655] Throttling request took 1.046428379s, request: GET:https://172.20.0.1:443/apis/policy/v1beta1?timeout=32s
2021-05-17T18:11:07.198Z	INFO	controller-runtime.metrics	metrics server is starting to listen	{"addr": "127.0.0.1:8080"}
2021-05-17T18:11:07.198Z	INFO	config	Loading YAML Configuration from secret
2021-05-17T18:11:07.216Z	INFO	monitors	Configuration added for StatusCake
2021-05-17T18:11:07.216Z	INFO	setup	starting manager
I0517 18:11:07.217617       1 leaderelection.go:243] attempting to acquire leader lease platform/<XX>.stakater.com...
2021-05-17T18:11:07.217Z	INFO	controller-runtime.manager	starting metrics server	{"path": "/metrics"}
I0517 18:11:24.673357       1 leaderelection.go:253] successfully acquired lease platform/<XX>.stakater.com
2021-05-17T18:11:24.673Z	DEBUG	controller-runtime.manager.events	Normal	{"object": {"kind":"ConfigMap","namespace":"platform","name":"<XX>.stakater.com","uid":"45a75228-33a7-400f-8dff-cf647272476a","apiVersion":"v1","resourceVersion":"384985618"}, "reason": "LeaderElection", "message": "platform-ingressmonitorcontroller-5b4ffb5c44-brw4b_f1ec386a-26f0-40f4-8743-130f58664a7a became leader"}
2021-05-17T18:11:24.673Z	DEBUG	controller-runtime.manager.events	Normal	{"object": {"kind":"Lease","namespace":"platform","name":"<XX>.stakater.com","uid":"90544dae-9fbd-4a0a-b613-3e5f99ecbc71","apiVersion":"coordination.k8s.io/v1","resourceVersion":"384985619"}, "reason": "LeaderElection", "message": "platform-ingressmonitorcontroller-5b4ffb5c44-brw4b_f1ec386a-26f0-40f4-8743-130f58664a7a became leader"}
2021-05-17T18:11:24.673Z	INFO	controller-runtime.manager.controller.endpointmonitor	Starting EventSource	{"reconciler group": "endpointmonitor.stakater.com", "reconciler kind": "EndpointMonitor", "source": "kind source: /, Kind="}
2021-05-17T18:11:24.774Z	INFO	controller-runtime.manager.controller.endpointmonitor	Starting Controller	{"reconciler group": "endpointmonitor.stakater.com", "reconciler kind": "EndpointMonitor"}
2021-05-17T18:11:24.774Z	INFO	controller-runtime.manager.controller.endpointmonitor	Starting workers	{"reconciler group": "endpointmonitor.stakater.com", "reconciler kind": "EndpointMonitor", "worker count": 1}
2021-05-17T18:11:25.421Z	DEBUG	statuscake-monitor	No data has been updated (is any data different?)
E0517 18:11:25.915483       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.2/tools/cache/reflector.go:167: Failed to watch *v1.Service: unknown (get services)
E0517 18:11:26.349594       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.2/tools/cache/reflector.go:167: Failed to watch *v1.Pod: unknown (get pods)
2021-05-17T18:11:26.831Z	DPANIC	statuscake-monitor	odd number of arguments passed as key-value pairs for logging	{"ignored key": "platform-thanos-bucketweb-platform"}
github.com/go-logr/zapr.handleFields
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:100
github.com/go-logr/zapr.(*zapLogger).Info
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:127
sigs.k8s.io/controller-runtime/pkg/log.(*DelegatingLogger).Info
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/log/deleg.go:130
github.com/stakater/IngressMonitorController/pkg/monitors/statuscake.(*StatusCakeMonitorService).Update
	/workspace/pkg/monitors/statuscake/statuscake-monitor.go:324
github.com/stakater/IngressMonitorController/pkg/monitors.(*MonitorServiceProxy).Update
	/workspace/pkg/monitors/monitor-proxy.go:96
github.com/stakater/IngressMonitorController/pkg/controllers.(*EndpointMonitorReconciler).handleUpdate
	/workspace/pkg/controllers/endpointmonitor_updated.go:26
github.com/stakater/IngressMonitorController/pkg/controllers.(*EndpointMonitorReconciler).Reconcile
	/workspace/pkg/controllers/endpointmonitor_controller.go:95
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:298
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:253
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.2
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:216
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:185
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:155
k8s.io/apimachinery/pkg/util/wait.BackoffUntil
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:156
k8s.io/apimachinery/pkg/util/wait.JitterUntil
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:133
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:185
k8s.io/apimachinery/pkg/util/wait.UntilWithContext
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:99
E0517 18:11:26.831258       1 runtime.go:76] Observed a panic: odd number of arguments passed as key-value pairs for logging
goroutine 457 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x174e5e0, 0xc003ebe010)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/runtime/runtime.go:74 +0x95
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/runtime/runtime.go:48 +0x86
panic(0x174e5e0, 0xc003ebe010)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0005feb00, 0xc0042b0600, 0x1, 0x1)
	/go/pkg/mod/go.uber.org/zap@v1.15.0/zapcore/entry.go:230 +0x565
go.uber.org/zap.(*Logger).DPanic(0xc0003bc0c0, 0x1a16f0c, 0x3d, 0xc0042b0600, 0x1, 0x1)
	/go/pkg/mod/go.uber.org/zap@v1.15.0/logger.go:215 +0x85
github.com/go-logr/zapr.handleFields(0xc0003bc0c0, 0xc002b8dfc0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x7f4c3e8a4c10, 0x8, 0x10)
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:100 +0x5e5
github.com/go-logr/zapr.(*zapLogger).Info(0xc0002fbf30, 0x19ca043, 0x10, 0xc002b8dfc0, 0x1, 0x1)
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:127 +0xb0
sigs.k8s.io/controller-runtime/pkg/log.(*DelegatingLogger).Info(0xc00012cc30, 0x19ca043, 0x10, 0xc002b8dfc0, 0x1, 0x1)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/log/deleg.go:130 +0xa2
github.com/stakater/IngressMonitorController/pkg/monitors/statuscake.(*StatusCakeMonitorService).Update(0xc000114230, 0xc002c08180, 0x39, 0xc0006a9e90, 0x22, 0xc0001811d0, 0x7, 0x17bfc20, 0xc0007f8160)
	/workspace/pkg/monitors/statuscake/statuscake-monitor.go:324 +0x63f
github.com/stakater/IngressMonitorController/pkg/monitors.(*MonitorServiceProxy).Update(...)
	/workspace/pkg/monitors/monitor-proxy.go:96
github.com/stakater/IngressMonitorController/pkg/controllers.(*EndpointMonitorReconciler).handleUpdate(0xc00003b400, 0xc00072f420, 0x8, 0xc0006ab0c0, 0x19, 0xc0004e0680, 0xc0004c2f00, 0x37, 0xc0006a9e90, 0x22, ...)
	/workspace/pkg/controllers/endpointmonitor_updated.go:26 +0x33a
github.com/stakater/IngressMonitorController/pkg/controllers.(*EndpointMonitorReconciler).Reconcile(0xc00003b400, 0x1bfb298, 0xc00067e5a0, 0xc00072f420, 0x8, 0xc0006ab0c0, 0x19, 0xc00067e5a0, 0xc000c62000, 0x185e840, ...)
	/workspace/pkg/controllers/endpointmonitor_controller.go:95 +0x54c
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc000c1bd60, 0x1bfb1f0, 0xc00003b540, 0x18186e0, 0xc00088e0c0)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:298 +0x30d
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc000c1bd60, 0x1bfb1f0, 0xc00003b540, 0x0)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:253 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.2(0x1bfb1f0, 0xc00003b540)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:216 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc0005e9f50)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc000b77f50, 0x1bc8000, 0xc000b63b90, 0xc00003b501, 0xc000384180)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:156 +0x9b
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc0005e9f50, 0x3b9aca00, 0x0, 0x1, 0xc000384180)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x1bfb1f0, 0xc00003b540, 0xc0008303e0, 0x3b9aca00, 0x0, 0x1)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x1bfb1f0, 0xc00003b540, 0xc0008303e0, 0x3b9aca00)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:213 +0x40d
panic: odd number of arguments passed as key-value pairs for logging [recovered]
	panic: odd number of arguments passed as key-value pairs for logging

goroutine 457 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/runtime/runtime.go:55 +0x109
panic(0x174e5e0, 0xc003ebe010)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0005feb00, 0xc0042b0600, 0x1, 0x1)
	/go/pkg/mod/go.uber.org/zap@v1.15.0/zapcore/entry.go:230 +0x565
go.uber.org/zap.(*Logger).DPanic(0xc0003bc0c0, 0x1a16f0c, 0x3d, 0xc0042b0600, 0x1, 0x1)
	/go/pkg/mod/go.uber.org/zap@v1.15.0/logger.go:215 +0x85
github.com/go-logr/zapr.handleFields(0xc0003bc0c0, 0xc002b8dfc0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x7f4c3e8a4c10, 0x8, 0x10)
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:100 +0x5e5
github.com/go-logr/zapr.(*zapLogger).Info(0xc0002fbf30, 0x19ca043, 0x10, 0xc002b8dfc0, 0x1, 0x1)
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:127 +0xb0
sigs.k8s.io/controller-runtime/pkg/log.(*DelegatingLogger).Info(0xc00012cc30, 0x19ca043, 0x10, 0xc002b8dfc0, 0x1, 0x1)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/log/deleg.go:130 +0xa2
github.com/stakater/IngressMonitorController/pkg/monitors/statuscake.(*StatusCakeMonitorService).Update(0xc000114230, 0xc002c08180, 0x39, 0xc0006a9e90, 0x22, 0xc0001811d0, 0x7, 0x17bfc20, 0xc0007f8160)
	/workspace/pkg/monitors/statuscake/statuscake-monitor.go:324 +0x63f
github.com/stakater/IngressMonitorController/pkg/monitors.(*MonitorServiceProxy).Update(...)
	/workspace/pkg/monitors/monitor-proxy.go:96
github.com/stakater/IngressMonitorController/pkg/controllers.(*EndpointMonitorReconciler).handleUpdate(0xc00003b400, 0xc00072f420, 0x8, 0xc0006ab0c0, 0x19, 0xc0004e0680, 0xc0004c2f00, 0x37, 0xc0006a9e90, 0x22, ...)
	/workspace/pkg/controllers/endpointmonitor_updated.go:26 +0x33a
github.com/stakater/IngressMonitorController/pkg/controllers.(*EndpointMonitorReconciler).Reconcile(0xc00003b400, 0x1bfb298, 0xc00067e5a0, 0xc00072f420, 0x8, 0xc0006ab0c0, 0x19, 0xc00067e5a0, 0xc000c62000, 0x185e840, ...)
	/workspace/pkg/controllers/endpointmonitor_controller.go:95 +0x54c
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc000c1bd60, 0x1bfb1f0, 0xc00003b540, 0x18186e0, 0xc00088e0c0)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:298 +0x30d
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc000c1bd60, 0x1bfb1f0, 0xc00003b540, 0x0)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:253 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.2(0x1bfb1f0, 0xc00003b540)
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:216 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc0005e9f50)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc000b77f50, 0x1bc8000, 0xc000b63b90, 0xc00003b501, 0xc000384180)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:156 +0x9b
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc0005e9f50, 0x3b9aca00, 0x0, 0x1, 0xc000384180)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x1bfb1f0, 0xc00003b540, 0xc0008303e0, 0x3b9aca00, 0x0, 0x1)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x1bfb1f0, 0xc00003b540, 0xc0008303e0, 0x3b9aca00)
	/go/pkg/mod/k8s.io/apimachinery@v0.20.2/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.3/pkg/internal/controller/controller.go:213 +0x40d
```

K8 Version: `v1.16.15-eks`

--- a/pkg/monitors/statuscake/statuscake-monitor.go
+++ b/pkg/monitors/statuscake/statuscake-monitor.go
@@ -321,7 +321,7 @@ func (service *StatusCakeMonitorService) Update(m models.Monitor) {
 			return
 		}
 		if fa.Success {
-			log.Info("Monitor Updated:" + m.Name)
+			log.Info("Monitor Updated: " + m.Name)
 		} else {
 			log.V(1).Info("Monitor couldn't be updated: " + m.Name)
 			log.V(1).Info(fa.Message)

--- a/pkg/monitors/statuscake/statuscake-monitor.go
+++ b/pkg/monitors/statuscake/statuscake-monitor.go
@@ -321,7 +321,7 @@ func (service *StatusCakeMonitorService) Update(m models.Monitor) {
 			return
 		}
 		if fa.Success {
-			log.Info("Monitor Updated:", m.Name)
+			log.Info("Monitor Updated:" + m.Name)
 		} else {
 			log.V(1).Info("Monitor couldn't be updated: " + m.Name)
 			log.V(1).Info(fa.Message)

